plugins/modules/global_parameter.py

```diff
@@ -141,6 +154,8 @@ def main():
             module.foreman_params['value'] = parameter_value_to_str(module.foreman_params['value'], module.foreman_params['parameter_type'])
             if entity and 'value' in entity:
                 entity['value'] = parameter_value_to_str(entity['value'], entity.get('parameter_type', 'string'))
+            if entity and 'hidden_value?' in entity:
+                entity['hidden_value'] = entity.pop('hidden_value?')
```

Comment:
this smells like we'll have a HiddenValueEntity at some point, as the same code also exists in smart_class_parameter ;)

Reply:
:nose: not today?

Reply:
absolutely, just wanted to raise awareness